modules/tinymce/src/plugins/autosave/main/ts/api/Settings.ts

```diff
@@ -13,14 +13,13 @@ const shouldAskBeforeUnload = (editor) => {
 };
 
 const getAutoSavePrefix = (editor) => {
-  let prefix = editor.getParam('autosave_prefix', 'tinymce-autosave-{path}{query}{hash}-{id}-');
+  const l = document.location;
```

Comment:
`loc` or `location` instead maybe? as single char variables are generally avoided since they are less readable. In saying that I haven't seen a hard rule about that and is more of an observation.

Reply:
> I have to wonder if something got ported over incorrectly from v4 though, given started wasn't used 🤔

It is a possibility, yes. I gave autosave a quick test and it seems to be working. 

The Cell appeared to be an attempt to stop the setInterval being set up twice, but I can't figure out how that would have even happened.

Reply:
> `loc` or `location` instead maybe? as single char variables are generally avoided since they are less readable. In saying that I haven't seen a hard rule about that and is more of an observation.

Done.

I've seen so many different opinions on this - even just at Tiny - that it all sort of just becomes a blur. :)

Reply:
> It is a possibility, yes. I gave autosave a quick test and it seems to be working.
> 
> The Cell appeared to be an attempt to stop the setInterval being set up twice, but I can't figure out how that would have even happened.

I looked at the v4 code and it was possible for it to run twice, as it was in `postRender` which was shared between the button and menu item. However in v5 it was moved to run on plugin init, instead of when the button/menu was first rendered. That does mean it's not needed anymore, so thanks again for catching that 👍

Reply:
Oh sweet. Thanks for the help spelunking!

Reply:
I did a quick debug build - I can only see it being registered once.